tests/test_integrated_channels/test_degreed/test_exporters/test_learner_data.py

```diff
@@ -73,11 +73,12 @@ def test_get_learner_data_record(self, completed_date, is_passing):
             course_id=self.course_id,
         )
         exporter = DegreedLearnerExporter('fake-user', self.config)
-        learner_data_record = exporter.get_learner_data_record(
+        learner_data_records = exporter.get_learner_data_records(
             enterprise_course_enrollment,
             completed_date=completed_date,
             is_passing=is_passing,
         )
+        learner_data_record = learner_data_records[0]
```

Comment:
Are we sure we're going to have at least one record here?  What if the array is empty for some reason?

Reply:
the method returns either None or an array with at least one item, changed from returning None or one item.

Reply:
makes sense to me